### ISSUE TITLE
Menu needs to be tab accessibile

### ISSUE THREAD

**Issue by muffindcc** [gantry/gantry5#2697]
If you are tabbing through the menu particle, dropdowns aren't triggered, which means someone who is using keyboard only navigation will be unable to access the additional menu items. Focus, as well as hover, should trigger the dropdown. This is quite a large accessibility issue.

**Comment by muffindcc** [comment]
For anyone that wants a quick fix, using jQuery:

```javascript
$('a.g-menu-item-container').focus(function(){

    var parents         = $(this).parents('.g-menu-item');
    var parents_reverse = parents.toArray().reverse();

    $.each(parents_reverse, function(i, o){
      if ($(this).hasClass('g-selected') == false) {
        $(this).find('> .g-menu-item-container > .g-menu-parent-indicator').click();
      }
    })
  })
```

**Comment by bayareajenn** [comment]
I agree that this would be really important to address. 

@jmshey90 thanks for the quick fix. I'm not a developer... I'm assuming I need to do some kind of override to get your quick fix to work. Where should I put it?

**Comment by jmshey90** [comment]
@bayareajenn we created a menu.js file in the template's /js folder, and added it via the CSS/JS Atom in Base Outline > Page Settings. As we've made a number of modifications to the default menu, its easier using a file. So you can do the same, or just use the CSS/JS Atom and add inline JS:

```
(function ($) {

  $('a.g-menu-item-container').focus(function(){

    var parents         = $(this).parents('.g-menu-item');
    var parents_reverse = parents.toArray().reverse();

    $.each(parents_reverse, function(i, o){
      if ($(this).hasClass('g-selected') == false) {
        $(this).find('> .g-menu-item-container > .g-menu-parent-indicator').click();
      }
    })
  })

})(jQuery)
```

**Comment by bayareajenn** [comment]
@jmshey90 thank you. Sadly, not working for me in J4. Maybe some classes changed and I need to dig into the code a bit. Thank you for your response! It was debatable after two years you would. Thank you!

**Comment by jmshey90** [comment]
I should specify this is when not using separators. It doesn't work with separators.

**Comment by bayareajenn** [comment]
lol, that's why! So are y'all just having a single article or cat blog as the parent? I still can't get it to fly even as a single article or cat blog. I just end up having to "enter" into the parent. No drop-down will display regardless if it's enabled in the particle settings to open on hover or not. It's ok. I'll just put a page of content there with links. Ideal? No. But I'm not a coder. Also could be the template is different. I'm using a RocketTheme template and not Helium or Hydrogen. Thanks again!
